src/FakeItEasy/Fake.cs

```diff
@@ -48,7 +48,7 @@ public static IEnumerable<ICompletedFakeObjectCall> GetCalls(object fakedObject)
         /// </summary>
         /// <param name="fakedObject">The faked object to clear the configuration of.</param>
         [SuppressMessage("Microsoft.Naming", "CA1720:IdentifiersShouldNotContainTypeNames", MessageId = "object", Justification = "The term fake object does not refer to the type System.Object.")]
-        [Obsolete("ClearConfiguration will be removed in version 8.0.0. Use Fake.Reset, or discard the fake and create a new one.")]
+        [Obsolete("ClearConfiguration will be removed in version 9.0.0. Use Fake.Reset, or discard the fake and create a new one.")]

```

Comment:
Not for this PR, but do we want an analyzer fixup rule to help migrate? It might be a teeny bit helpful, but the "obsolete" message is pretty clear and easy to act on. Let's say no!

Reply:
Interesting idea. What kind of analyzer do you have in mind? What would it do exactly? Check if the current version is >= than the version mentioned in the message?

Reply:
It would have to see a usage of `ClearConfiguration` and suggest (if the current FIE is one that has `Reset`) the rewrite to `Reset`. It wouldn't necessarily be based on the Obsolete message itself. It could work solely based on the source code (and FIE version, assuming we can get that, which I bet we can).

But I think it's of limited value.

Reply:
Ah, OK. I thought you meant an analyzer so that we don't forget to remove things we said we would remove 😆 
Yeah, probably not worth the trouble.